src/main/java/com/autotune/analyzer/utils/AnalyzerErrorConstants.java

```diff
@@ -193,6 +193,10 @@ public static final class CreateExperimentAPI {
             public static final String INVALID_MODEL_NAME = " model name is not supported. Use cost or performance.";
             public static final String MULTIPLE_MODELS_UNSUPPORTED = "Multiple models are currently not supported for auto or recreate mode.";
             public static final String WHITESPACE_NOT_ALLOWED = "Whitespace can not be entered as a term or model value ";
+            public static final String MISSING_NAMESPACE_DATA = "Missing NamespaceData for %s experimentType";
+            public static final String MISSING_NAMESPACE_NAME = "Missing `namespace_name` for %s experimentType";
```

Comment:
Frame both of these to something like this:

`Missing `namespace_name` for experimentType: %s`